spiceaidocs/docs/api/auth/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Authentication'
+sidebar_label: 'Authentication'
+sidebar_position: 3
+description: 'Authentication documentation'
+pagination_prev: null
+pagination_next: null
+---
+
+Spice supports adding optional authentication to its API endpoints via configurable API keys.
+
+Use the `auth` section as a child to `runtime` to provide the API keys. Multiple API keys can be specified, and any of the keys can be used to authenticate requests.
+
+```yaml
+runtime:
+  auth:
+    api-key:
+      enabled: true
+      keys:
```

Comment:
Does this work?
```yaml
keys: ${ secrets:api_key_list }
```
`export api_key_list="['foo','bar']"`

Reply:
No